### ISSUE TITLE
[Feature]: Disable Vue Devtools plugin during tests

### ISSUE THREAD

**Issue by samuelstein** [microsoft/playwright#35398]
### 🚀 Feature Request

 I experienced a problem during accessibility testing with playwright and axe (https://www.npmjs.com/package/@axe-core/playwright):
The test and accessibility scan always included the vue dev tools which is not intended or should be at least configurable. 
Would be nice if there will be a config option to disable vue dev tools or at least the selector '.ignore-axe' will be added to the vue dev tools button.

### Example

I worked around this behavior with overwriting the vue devtools hook like this:
`test.beforeEach(async ({ page }) => {
  // Disable Vue Devtools
  await page.addInitScript(() => {
    Object.defineProperty(window, '__VUE_DEVTOOLS_GLOBAL_HOOK__', {
      value: undefined,
      writable: true,
    })
  })
})`

### Motivation

This makes the tests faster and results in less false positives.

**Comment by mxschmitt** [comment]
This issue looks like it was filed int he wrong repository. Probably https://github.com/dequelabs/axe-core-npm is better.

**Comment by samuelstein** [comment]
I can try but isn't playwright the platform who executes the tests and therefore could allow configuration?
